### DISCUSSION TITLE
RTC spoof without internet

### DISCUSSION THREAD

**Discussion by NikoHuerta** [Rparadise-Team/Koriki#33] [Ideas & Requests]
Miyoo Mini doesn't have an internal RTC.  We can emulate this function, saving the shutdown time  and increasing the hours in a seteable value on next system boot. 

What do you guys think about it?. With a little support from someone who knows better the repo, i can adventure to do the implementation.

Thanks in advance

**Comment by NikoHuerta** [comment]
Edited msg!

**Comment by Ninoh-FOX** [comment]
Fixed
